### ISSUE TITLE
No module named 'scipy'

### ISSUE THREAD

**Issue by Goran-L** [dmnfarrell/tablexplore#7]
Hi,
first I wanted to thank you for the amazing work you did with Tablexplore. 
I've been testing the latest version and I think I found some bugs.
I could reproduce it on Windows 7 and 10:

If I choose plot type: density and axes layout multiple
Tablexplore will throw an error 

=> No module named 'scipy'

The Log Windows Command doesn't show any supplemental info about the error.

**Comment by dmnfarrell** [comment]
Thanks for the feedback. I didn't realise scipy was required for the density plots. I usually have it installed so didn't notice. I may remove this plot type as I do not currently want scipy as a dependency.

**Comment by Goran-L** [comment]
I guess that it would increase a lot the size of the standalone version.

**Comment by dmnfarrell** [comment]
Yes it might, but I can try it and see.

**Comment by Goran-L** [comment]
Hi,
I just checked the standalone 0.51 and it throws new error at startup:
'failed to load gis plugin
No module named 'geopandas'.
Nevertheless it doesn't seem to be critical.

By the way, the demo plot test plugin is really awesome. It's much easier to pick the most appropriate figure display.

**Comment by dmnfarrell** [comment]
Yes you can ignore this message. Some plugins that require other libraries won't be included in the windows binary.
